api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/controllers/api/StaticAPIRefreshController.java

```diff
@@ -18,18 +18,18 @@
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
-import org.zowe.apiml.apicatalog.staticapi.StaticAPIService;
+import org.zowe.apiml.apicatalog.staticapi.StaticService;
 import reactor.core.publisher.Mono;
 import reactor.core.scheduler.Schedulers;
 
 @RequiredArgsConstructor
 public class StaticAPIRefreshController {
 
-    private final StaticAPIService staticAPIService;
+    private final StaticService staticService;
```

Comment:
maybe StaticRegistrationService? StaticService sounds like it's missing something

Reply:
renamed